test/libsolidity/semanticTests/userDefinedValueType/abicodec.sol

```diff
@@ -0,0 +1,36 @@
+// A test to see if `abi.encodeWithSelector(..., (CustomType))` works as intended.
+contract C {
+    type MyInt is int;
+    function f(MyInt x) external returns(MyInt a, MyInt b, MyInt c, MyInt d) {
+        a = MyInt.wrap(-1);
+        b = MyInt.wrap(0);
+        c = MyInt.wrap(1);
+        d = x;
+    }
+    function g() external returns(bool) {
+        (bool success1, bytes memory ret1) =  address(this).call(abi.encodeWithSelector(this.f.selector, MyInt.wrap(5)));
```

Comment:
Why does `abi.encodeWithSelector` work with both `MyInt` and `int`?

Reply:
Why is that an issue?

Reply:
Ah, we don't do any type checking, right?